File: 151001/Verghel/DC3/publicator/src/main/java/com/example/publicator/dto/CreatorResponseTo.java
package com.example.publicator.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CreatorResponseTo {
    int id;
    String login;
    String password;
    String firstname;
    String lastname;
}

File: 151003/Alekseeva/TaskNoSQL/Module.Publisher/Dto/Response/CreatorResponseTo.cs
using System.Text.Json.Serialization;
namespace Publisher.Dto.Response;

public record CreatorResponseTo(
    [property: JsonPropertyName("id")] long Id,
    [property: JsonPropertyName("login")] string Login,
    [property: JsonPropertyName("password")]
    string Password,
    [property: JsonPropertyName("firstname")]
    string FirstName,
    [property: JsonPropertyName("lastname")]
    string LastName
);